store/copr/mpp.go

```diff
@@ -343,7 +343,7 @@ func (m *mppIterator) establishMPPConns(bo *backoffer, req *kv.MPPDispatchReques
 				return
 			}
 
-			if err1 := bo.Backoff(tikv.BoTiKVRPC, errors.Errorf("recv stream response error: %v", err)); err1 != nil {
+			if err1 := bo.b.BackoffTiKVRPC(errors.Errorf("recv stream response error: %v", err)); err1 != nil {
```

Comment:
Note: I believe it's actually a bug for TiFlash requests to use TiKV backoff. This will make the timeout error message to be "TiKV server timeout" which is incorrect.
(needn't be fixed in this pr)

Reply:
Yes,you are right